docs/upgrading-to-datastreams.asciidoc

```diff
@@ -0,0 +1,91 @@
+[[upgrade-to-data-streams]]
+=== Upgrade to data streams
+
+* <<why-upgrade-to-data-streams>>
+* <<considerations-data-streams>>
+* <<apm-data-streams-upgrade-steps>>
+
+Classic indices are deprecated starting in version 7.16.0.
+Users that are not ready to <<upgrade-to-apm-integration>> should
+upgrade to data streams in the meantime.
```

Comment:
We'll be updating these docs again after 7.16.0 is out, right? Data streams are required in 8.0.0, and the `apm-server.data_streams.enabled` config mentioned further down has been removed.

Reply:
That's correct. These will be revised again for 8.0 🙃